config/initializers/paper_trail.rb

```diff
@@ -1,3 +1,3 @@
 # frozen_string_literal: true
 
-PaperTrail.config.track_associations = false
\ No newline at end of file
+# PaperTrail.config.track_associations = false
```

Comment:
Why is this commented out? Should we just remove this file?

Reply:
Ok Stephen